topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -257,18 +261,48 @@ TAD calling works in two steps: First HiCExplorer computes a TAD-separation scor
 
 As an output we get the boundaries, domains and scores separated files. We will use in the plot later only the TAD-score file.
 
+# A/B compartments computation
 
 > ### {% icon hands_on %} Hands-on: Computing A / B compartments
 >
 > 1. **hicPCA** {% icon tool %}: Run hicPCA adjusting the parameters:
 >    - "Matrix to compute on" to `corrected contact matrix dm3 large`
 >    - "Output file format" to `bigwig`
+>    - "Return internally used Pearson matrix" to `Yes`
+>
+{: .hands_on}
+
+> ### {% icon hands_on %} Hands-on: Plotting the pearson matrix and PCA track
+>
+> 1. **hicPlotMatrix** {% icon tool %}: Run hicPlotMatrix on `pearson_matrix from PCA computation` adjusting the parameters:
+>    - "Plot title" to `Pearson matrix and PC1`
+>    - "Chromosomes to include" to `chr2L`
+>    - "Color map to use for the heatmap" to `gist_heat`
+>    - "Datatype of eigenvector file" to `bigwig`
+>    - "Eigenvector file" to `hicPCA on [...] PC1`
 >
 {: .hands_on}
 
+> 1. **hicPlotMatrix** {% icon tool %}: Run hicPlotMatrix on `pearson_matrix from PCA computation` adjusting the parameters:
+>    - "Plot title" to `Pearson matrix and PC2`
+>    - "Chromosomes to include" to `chr2L`
+>    - "Color map to use for the heatmap" to `gist_heat`
+>    - "Datatype of eigenvector file" to `bigwig`
+>    - "Eigenvector file" to `hicPCA on [...] PC2`
+>
+{: .hands_on}
+
+
+![Pearson PC1](../../images/pearson_pc1.png)
+
+
+![Pearson PC2](../../images/pearson_pc2.png)
+
+The first principal component correlates with the chromosome arms, while the second component correlates with A/B compartments. 
+
 # Integrating Hi-C and other data
 
-We can plot the TADs for a given chromosomal region. For this we will use [hicPlotTADs](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicPlotTADs.html).
+We can plot the TADs for a given chromosomal region. For this we will use [pyGenomeTracks](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicPlotTADs.html).

```

Comment:
the link should point to pyGenomeTracks?

Reply:
No, in the documentation of HiCExplorer it is still hicPlotTads, and the readthedocs of pgt is not public at the moment.